app/javascript/components/patient/assessment/steps/SymptomsAssessment.js

```diff
@@ -234,17 +243,41 @@ class SymptomsAssessment extends React.Component {
   render() {
     return (
       <Card className="mx-0 card-square">
-        <Card.Header className="h4">
-          {this.props.translations[this.props.lang]['html']['weblink']['title']}{' '}
-          {this.props.patient_initials && this.props.patient_age !== null && (
-            <span>
-              ({this.props.patient_initials}-{this.props.patient_age})
-            </span>
-          )}
-          {this.props.patient_initials && this.props.patient_age === null && <span>({this.props.patient_initials})</span>}
-          {!this.props.patient_initials && this.props.patient_age !== null && <span>({this.props.patient_age})</span>}
-        </Card.Header>
+        {!this.props.current_user && (
+          // If this report is being edit by a user, assume header content rendered at a higher level
+          <Card.Header className="h4">
+            {this.props.translations[this.props.lang]['html']['weblink']['title']}{' '}
+            {this.props.patient_initials && this.props.patient_age !== null && (
+              <span>
+                ({this.props.patient_initials}-{this.props.patient_age})
+              </span>
+            )}
+            {this.props.patient_initials && this.props.patient_age === null && <span>({this.props.patient_initials})</span>}
+            {!this.props.patient_initials && this.props.patient_age !== null && <span>({this.props.patient_age})</span>}
+          </Card.Header>
+        )}
         <Card.Body>
+          {this.props.current_user && (
+            // Only allow users, not monitorees, to edit the report date
+            <Form.Group>
+              <Form.Row>
+                <Form.Label className="input-label mr-2 pt-1">Symptom Report for Date:</Form.Label>
+                <DateInput
+                  id="reported_at"
+                  date={this.props.assessment.reported_at ?? moment().toDate()}
+                  minDate={'2020-01-01'}
+                  maxDate={moment()}
+                  onChange={this.handleDateChange}
+                  placement="bottom"
+                  isInvalid={false}
+                  customClass="form-control-sm"
+                  ariaLabel="Assessment Reported Date Input"
+                  showTime={true}
+                />
+                <InfoTooltip tooltipTextKey={'reportedAtTime'} />
```

Comment:
I would vertically center the info tooltip here and maybe give a little more left padding

<img width="400" alt="Screen Shot 2021-10-21 at 11 19 50 AM" src="https://user-images.githubusercontent.com/35042815/138308211-61d511a5-1f3d-49f9-953d-4c6a811c4553.png">

Reply:
I know this is more work, but I'd prefer to see Info tooltip taking a custom class then just wrapping this in a div.  Also, if it's possible to use the `align-middle` bootstrap class instead of hardcoding padding that would be preferred